x-pack/agent/docs/agent_configuration_example.yml

```diff
@@ -0,0 +1,172 @@
+outputs:
+  default:
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+    # Not supported at first
+    queue:
+      type: disk
+  long_term_storage:
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+    queue:
+      type: disk
+
+  monitoring:
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+
+settings.monitoring:
+  use_output: monitoring
+
+datasources:
+  # use the nginx package
+  - id: nginx-x1
+    title: "This is a nice title for human"
+    # @Ruflin: This is something that Jen raised here, how do I know where that datasource is managed?
+    # Is this something handled by fleet where we have an association between the ID and the creator
+    # of the datasource.
+    managed_by: fleet
+    # Package this config group is coming from. On importing, we know where it belongs
```

Comment:
@ruflin the link between the datasource and the "managing" app was mentioned by @jen-huang, I think we wanted to make the link hidden using the ids but maybe making it explicit in the configuration is a good idea.

Reply:
@jen-huang  Had a chat with @ruflin and we do not need to have a `managed_by` defined in the datasource, actually that information should be in the package. This make sense if lets say we start making custom UI for specific datasources. Maybe a better UI for AWS services.

Reply:
@ph So datasources created by Uptime, APM, Endpoint, etc will all specify a package? What if they don't use a package?

Reply:
Yes. They will have to have a package even if it is only to define this linking. Otherwise we could hardcode it into our UI as these links should not change too often.